portfolio/src/main/java/com/google/sps/helper/ValidationResult.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
```

Comment:
Interesting that you wrote this class. It reminds me a lot of a C++ utility we have called StatusOr (I think it's not quite [released externally yet](https://github.com/google/cel-cpp/issues/54)). But for example here's some [documentation on it](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/stubs/statusor.h) if you want to take a look at how they documented it.

Reply:
Oh cool! It also kind of reminds me of [std::optional](https://en.cppreference.com/w/cpp/utility/optional) in C++ as well.